nubis/terraform/main.tf

```diff
@@ -173,3 +173,14 @@ module "backup" {
   purpose      = "backup"
   role         = "${module.worker.role}"
 }
+
+module "cloudfront" {
+  source                 = "github.com/nubisproject/nubis-terraform//cloudfront?ref=131daeedc94660a39fb8de148706070c6d293cf2"
+  region                 = "${var.region}"
+  environment            = "${var.environment}"
+  account                = "${var.account}"
+  service_name           = "${var.service_name}"
+  domain_aliases         = ["www.publicsuffix.org", "publicsuffix.org"]
```

Comment:
A few issues here for me.

- This sticks site-specific domain names in main.tf, when publicsuffix really lives under nubis/terraform/sites.tf
- The primary domain name 'publicsuffix.org' is duplicated twice, once for domain_aliases and once for acm_certificate_domain. Feels to me like it would be cleaner to do something like

```toml
<pseudo code>
domains = [ 'publicsuffix.org','www.publicsuffix.org']
domain_aliases = "${domains}"
acm_certificate_domain = "${domains[0]}"
</pseudo code>
- This doesn't distinguish between stage/prod in any way